commit a9cb8effcc7ae63e42373d559842e06a146f9e34 (refs/changes/77/5577/2)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2019-02-15T15:00:51+00:00 (8 months ago)
    
    TRITON-1220 node-tap versions between 12.0.1 and 12.5.2 can emit invalid TAP
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/package.json b/package.json
index 5dc91f5..63b5be0 100644
--- a/package.json
+++ b/package.json
@@ -36,6 +36,6 @@
     "restify": "^7.0.0",
     "restify-clients": "^2.6.4",
     "restify-errors": "^5.0.0",
-    "tap": "^12.1.1"
+    "tap": "^12.5.2"
   }
 }
